### ISSUE TITLE
Lua Error in 10.0

### ISSUE THREAD

**Issue by aa-chrismcfadyen** [dandruff/xCT#213]
```
1x Interface/AddOns/xCT+/modules/combattext.lua:877: attempt to call method 'CombatStateChanged' (a nil value)
[string "@Interface/AddOns/xCT+/modules/combattext.lua"]:877: in function `?'
[string "@Interface/AddOns/xCT+/modules/combattext.lua"]:565: in function <Interface/AddOns/xCT+/modules/combattext.lua:540>
```

Happened on first combat

**Comment by Daeveren** [comment]
2 other errors:
`Message: Interface/AddOns/xCT+/modules/blizzard.lua:39: attempt to index global 'InterfaceOptionsCombatPanel' (a nil value)
Time: Wed Oct 26 20:24:56 2022
Count: 1
Stack: Interface/AddOns/xCT+/modules/blizzard.lua:39: attempt to index global 'InterfaceOptionsCombatPanel' (a nil value)
[string "=[C]"]: ?
[string "@Interface/AddOns/xCT+/modules/blizzard.lua"]:39: in main chunk

Locals: `

`Message: Interface/AddOns/xCT+/core.lua:1709: AceConfigRegistry-3.0: RegisterOptionsTable(appName, options): 'options' - expected table or function reference
Time: Wed Oct 26 20:24:56 2022
Count: 1
Stack: Interface/AddOns/xCT+/core.lua:1709: AceConfigRegistry-3.0: RegisterOptionsTable(appName, options): 'options' - expected table or function reference
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "@Interface/AddOns/Ace3/AceConfig-3.0/AceConfig-3.0.lua"]:47: in function `RegisterOptionsTable'
[string "@Interface/AddOns/xCT+/core.lua"]:1709: in main chunk

Locals: `

**Comment by jgrull** [comment]
No LUA errors, but combat text is not displaying, and /xct in game command does not pull up config

**Comment by Yunzlez** [comment]
Not an addon developer - but I managed to get it to at least work on 10.0.0.

I haven't tested in-depth, but as far as I can tell everything works, including the settings panel.

Pretty sure the merging feature won't work for any new abilities, or anything evoker though.

I also renamed the fork to xCT++, to be able to install it side-by-side with xCT+, for when xCT+ is updated in the future :) 
https://github.com/Yunzlez/xCT/releases/tag/10.0-hotfix

Make sure to disable xCT+ before using this. It should just read your settings.
